instruqt/avoiding-installation-pitfalls/05-validating-before-an-install/check-shell

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# This set line ensures that all failures will cause the script to error and exit
+set -euxo pipefail
+source /etc/profile.d/header.sh
+
+result=0
+
+# check for release to Unstable
+api_token=$(get_api_token)
+customer_email=${INSTRUQT_PARTICIPANT_ID}@geeglo.io
```

Comment:
Unless I missed it, I think this is missing from the instructions. Wouldn't have figured this out without looking at the code 😄 . Unless the email needs to be unique for each lab, it probably can just be static?

Reply:
For clarity, I was failing the check in this stage, because I was not using a customer email that matched this expected value.

Reply:
I may not have remembered it in this lab. Nice catch.

Reply:
I did find it in there. Making it more prominent.